server/package.json

```diff
@@ -21,7 +22,7 @@
     "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@prisma/client": "^5.5.2",
+    "@prisma/client": "^5.5.0",
```

Comment:
Is there any particular reason that we're downgrading the prisma client (this and the one on line 63)?

Reply:
Hmmm, I don't know how that change happened- must have been an accident. The only thing I meant to change in that file was adding the deployment script expand on nodes.js's default memory limit of 1.5gb. 
Good catch!

Reply:
Fixed in most recent push!